app/src/main/java/de/westnordost/streetcomplete/quests/bus_stop_bench/AddBenchStatusOnBusStop.kt

```diff
@@ -0,0 +1,44 @@
+package de.westnordost.streetcomplete.quests.bus_stop_bench
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.osm.osmquest.SimpleOverpassQuestType
+import de.westnordost.streetcomplete.data.osm.changes.StringMapChangesBuilder
+import de.westnordost.streetcomplete.data.osm.mapdata.OverpassMapDataAndGeometryApi
+import de.westnordost.streetcomplete.quests.YesNoQuestAnswerFragment
+import de.westnordost.streetcomplete.settings.ResurveyIntervalsStore
+
+class AddBenchStatusOnBusStop(o: OverpassMapDataAndGeometryApi, r: ResurveyIntervalsStore) : SimpleOverpassQuestType<Boolean>(o) {
+
+    override val tagFilters = """
+        nodes with
+        (
+          (public_transport = platform and ~bus|trolleybus|tram ~ yes)
+          or
+          (highway = bus_stop and public_transport != stop_position)
+        )
+        and physically_present != no and naptan:BusStopType != HAR
+        and (!bench or bench older today -${r * 5} years)
```

Comment:
```suggestion
        and (!bench or bench older today -${r * 4} years)
```

(All these are using 2, 4, 8,... unless there is no good reason for an exception, let's stay with this IMO)